recipes/lora_finetune_distributed.py

```diff
@@ -546,6 +548,20 @@ def _setup_model(
         # synchronize before training begins
         torch.distributed.barrier()
 
+        # Calculate model FLOPs for MFU tracking
+        if self._is_rank_zero:
+            # Calculate model FLOPs for MFU tracking
+            self._model_flops = mfu.get_transformer_flops(
+                hidden_size=cfg_model.hidden_size,
+                intermediate_size=cfg_model.hidden_size * 4,  # Standard transformer uses 4x
+                num_layers=cfg_model.num_layers,
```

Comment:
is this a fair generalization, e.g. hidden_size*4? I am not sure how this would work for more complex models, like llama 3.2 vision, that has encoder+decoder. 

We are also a bit afraid of the added complexity/ number of lines of code that mfu may introduce. If we can reduce it, it will be easier to approve the pr.

But i understand that here its just one line + multiple arguments.